svea_vision/launch/person_state_estimation_validation.launch

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+
+<launch>
+
+    <!-- Args -->
+    <arg name="rosbag_prefix"   default="/svea_ws/src/svea_vision/reprocessed_bags/reprocessed"/>
```

Comment:
If this launch file is meant for testing it makes sense of having rosbag here. Though, in that case, add argument to enable/disable rosbag (eg w/ `use_rosbag` or `start_rosbag`).

Reply:
Also in that case, name the launch file to something *_eval